cwf/gateway/deploy/cwag.yml

```diff
@@ -20,6 +20,7 @@
       vars:
         distribution: "bionic"
         repo: "cwf-prod"
+        preburn: true
```

Comment:
Why do we need to change this file?

Reply:
The `pkgrepo` Ansible role is also used in the CWAG VMs. They don't have a preburn step since they are created from a bare Ubuntu box, but all these Ansible steps now have been moved to preburn. That's why I'm introducing the preburn variable here. The CWAG tests are running on my branch now to check if that works.

Reply:
I don't think that's necessary, as it will still just use the Ubuntu base image.